### PR TITLE
Dev branch 

### PR DESCRIPTION
**Pull request by Christabelle143**
# In this project my partner and I;
-  Developed the Space Travellers' Hub following the Kanban board tasks.
- Fix linters errors
- Added descriptive README.md file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,68 @@
-# react-group-project
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+React-Redux Group Project: Space Travelers' Hub
+
+> React/Redux pair-programming project.
+
+
```
Comment (BenMukebo) [inline README.md]: - [ ] Please, add a meaningful  `Project description`. Remember A project description is a high-level overview of why you're doing a project. The document explains a project's objectives and essential qualities. Is an essential document that provides your audience with a comprehensive overview of the project. Think of it as the elevator pitch that focuses on what and why without delving into how. 😉 💯

File: README.md [hunk 2]
```diff
@@ -1 +1,68 @@
...
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Microverse
+
+## 📝 License
+
+This project is [MIT]() licensed.
```
Comment (BenMukebo) [inline README.md]: - [ ] When I click the link to the `MIT.md` file that you added, I get this 404 `error`:

00Mit


#### The reason for this is that there is no `MIT.md` file in the root directory of this project. Kindly attach the file. 👍

File: src/components/navbar.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+      <ul className="nav-links">
+        <li>
+          <NavLink className="nav-item" to="/rockets">Rockets</NavLink>
+        </li>
+        <li>
+          <NavLink className="nav-item" to="/missions">Missions</NavLink>
+        </li>
+        <li>
+          <NavLink className="nav-item" to="/profile">My Profile</NavLink>
+        </li>
```
Comment (BenMukebo) [inline src/components/navbar.js]: - [ ] Great job so far 👏🏻 But, Rockets should have the default view, so its data would be fetched when the application starts. However, the mission data should only be fetched (once) when a user navigates to the Missions section. You can achieve this by giving the root path to be the default value of your mission 🤔 

eg: `<NavLink className="nav-item" exact  to="/">Missions</NavLink>`

File: src/components/profile.js [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+          <div className="Content">
+            {
+              !reservedList.length
+                ? <p>No reservation!</p>
+                : reservedList.map((rocket) => <p key={rocket.id}>{rocket.rocket_name}</p>)
+            }
+          </div>
+        </div>
+      </div>
+    </div>
```
Comment (BenMukebo) [inline src/components/profile.js:43]: - [OPTIONAL] The `My Profile` section can be enhanced by adding the "Cancel reservation" and "Leave Mission" buttons to the lists here. Clicking them should dispatch the actions you have already used in the main Rockets and Missions sections.
- [OPTIONAL] The `My Profile` section can be enhanced by adding the "Read more" button for each mission and rocket. Upon clicking, it should open a corresponding Wikipedia page in a new tab. NOTE - you need to get that extra Wikipedia URL from the API's payload.

**Review by BenMukebo — CHANGES_REQUESTED** [review]
## Hi @Christabelle143, @Ceemos96, 

You did a great job so far 👏🏼 👏🏼
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


### To Highlight 🎉
- Git flow is followed. :heavy_check_mark:
- You have followed  JavaScript best practices. ✔️ 
- PR has a good title and summary. ✔️
- Routes and view components has been well created. ✔️ 
- `useSelector()` was used to get a state slice corresponding to routes. ✔️ 
- Your app has been well deployed using to Netlify. ✔️ 
- Your Design looks awesome, keep rocking :rocket: 

## STATUS: CHANGES REQUESTED  ♻️ 
_Please Check the comments under the review and make all required changes_
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers :clinking_glasses:  and Happy coding!!! :dancers:💻  

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me @BenMukebo when you comment so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/profile.js (hunk 4 above)
Comment (Christabelle143) [reply src/components/profile.js:43]: Thanks for review

File: README.md [hunk 5]
```diff
@@ -64,4 +64,4 @@ Give a ⭐️ if you like this project!
 
 ## 📝 License
 
-This project is [MIT]() licensed.
\ No newline at end of file
+This project is [MIT](https://github.com/Christabelle143/react-group-project/blob/development/LICENSE) licensed.
```
Comment (elmar8287) [inline README.md:67]: - [ ] OPTIONAL Please, consider that the link to the MIT.md file is still broken. To fix it, just use this correct link `https://github.com/Christabelle143/react-group-project/blob/dev/MIT.md`

image

**Review by elmar8287 — APPROVED** [review]
Hi, 

# APPROVED  👍 🥇 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

Congratulations! 🎉 

### Highlights

- Good PR description 👍 
- Nice README file 🥇 
- Good commits history 👍 

### Optional suggestions

- Please, consider my OPTIONAL suggestion under the review, before you will merge this PR! 👍 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
### Please, remember to tag me (@elmar8287) in your question so I can receive the notification 👍 
giphy

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**